client/my-sites/posts/style.scss

```diff
@@ -619,6 +623,7 @@
 	flex-grow: 1;
 	margin: 0 0 24px 24px;
 	display: none;
+	max-width: 400px;
```

Comment:
If you remove that at smaller sizes the drafts column tends to get very big, and seems a better balance if that gets limited at `400px`.